Makefile.docker

```diff
@@ -28,7 +28,7 @@ lint:
 	${CDWD} && go fmt ${NO_VENDOR}
 	golint ${SRC}
 
-test:
+test: vendor
```

Comment:
This is triggering a `godep restore` for me on every test run; the vendor folder from the host should be available from `DOCKERMAKE` in the main makefile. If we are going to assert this (which seems reasonable) it should be done in the main makefile's `test` target.